packages/dd-trace/test/appsec/rasp/command_injection.integration.spec.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const { createSandbox, FakeAgent, spawnProc } = require('../../../../../integration-tests/helpers')
+const getPort = require('get-port')
+const path = require('path')
+const Axios = require('axios')
+const { assert } = require('chai')
+
+// These test are here and not in the integration tests
+// because they require postgres instance
+describe('RASP - command_injection - integration', () => {
+  let axios, sandbox, cwd, appPort, appFile, agent, proc
+
+  before(async function () {
+    this.timeout(60000)
```

Comment:
is it necessary to wait 60 seconds in the `before` and `after`? is it not too much?

Reply:
Most of the integration test has this timeout, these tests should copy app to temp dir, install dependencies with `npm install` (or yarn), start the server... I think that this is a safe number to prevent flaky tests and if this goes faster, it is not going to be waiting, it is a max timeout, not a `setTimeout` to make this long.